crates/acir/src/circuit/gate.rs

```diff
@@ -64,7 +64,7 @@ impl std::fmt::Debug for Gate {
                 result = format!("x{} is {} bits", w.witness_index(), s);
             }
             Gate::Directive(Directive::Invert { x, result: r }) => {
-                result = format!("1/{}={}, or 0", x.witness_index(), r.witness_index());
+                result = format!("x{}=1/x{}, or 0", r.witness_index(), x.witness_index());
```

Comment:
Are these arguments in the right order? It seems it should be x first and then the inverse

Reply:
No it is fine, both ways are correct because 1/x=y <=> 1/y=x
Since we compute the inverse of let's say x, it is easier to read if we write r=1/x

Reply:
I agree with your logic, it seems to be the other way though?

If I have the value 5 and I want to compute 1/5 the format seems to be doing "x(1/5)=1/x(5)" So the first component is the inverse and the second component is the original value.

Reply:
So we have witness i equal to 5 and we want its inverse, so the directive takes a new witness j corresponding to the inverse.
Then we write x_j = 1/x_i, corresponding to x_j=1/5
I think we both agree but I fail to see your issue?

Reply:
Oh I see, the way its laid out is a bit confusing, but its not a blocker for me